packages/edit-post/src/editor.js

```diff
@@ -13,7 +13,7 @@ import { useMemo } from '@wordpress/element';
 import { SlotFillProvider } from '@wordpress/components';
 import { store as coreStore } from '@wordpress/core-data';
 import { store as preferencesStore } from '@wordpress/preferences';
-import { CommandMenu } from '@wordpress/commands';
+// eslint-disable-next-line no-unused-vars
 import { privateApis as coreCommandsPrivateApis } from '@wordpress/core-commands';
```

Comment:
If it's not used, just remove it no?

Reply:
@youknowriad yes I tried that but it didn't work. It needs the import to work or the commands do not get loaded at all. Not sure how to explain it still.

Reply:
Removed in [2873f18](https://github.com/WordPress/gutenberg/pull/54515/commits/2873f18de2b099f047274eca2d754fa7144c89a0)